File: JavaSamples/src/com/runninmancoding/bookSamples/classes/residences/Residence.java
package com.runninmancoding.bookSamples.classes.residences;

public class Residence {
	private int numRooms;

	public int getNumRooms() {
		return numRooms;
	}

	public void setNumRooms(int numRooms) {
		this.numRooms = numRooms;
	}
	
	protected void protectedResidenceMethod(){
		System.out.println("Protected method in Residence");
	}
	
	public void randomMessage(){
		System.out.println("Random message from Residence class");
	}

}

File: CSSamples/Residences/Residence.cs
using System;

namespace csBookSamples.Residences
{
    class Residence
    {
        private int numRooms;

        public int NumRooms { get => numRooms; set => numRooms = value; }

        public virtual void message()
        {
            Console.WriteLine("Message from Residence class");
        }

        protected void protectedMethod()
        {
            Console.WriteLine("Protected method in Residence class");
        }
    }
}
